readded everything (#104)
MrLongbottom/P10-Article: sections/appendix/authors.tex

\subsubsection{Author}\label{subsec:appendix_author}
There are a total of $227$ authors within the dataset where each author on average have written $757$ articles from 2017 to 2019.

> An interesting fact is that the median is $316$ which is much lower than the average which is visible in \autoref{fig:author_histogram}.

Reviewer: fix ref

> An interesting fact is that the median is $316$ which is much lower than the average which is visible in \autoref{fig:author_histogram}.

Reviewer 2: ```suggestion
An interesting fact is that the median is $323$ which is much lower than the average which is visible in \autoref{fig:auhtor_box_after}.
```

> This shows that while most authors have written just a few hundred articles, there is a small amount of authors that have written thousands of articles, increasing the average.

Reviewer: ```suggestion
This shows that while most authors have written just a few hundred articles, there is a small number of authors that have written thousands of articles, increasing the average.
```

The minimum number of articles that have been written by an author is $1$ and the maximum number is $9906$.
When we look at the top two most writing authors, we get these:
\begin{itemize}
	\item Ove Nørhave (9893 articles)
	\begin{itemize}

> 		\item A well-known journalist, who have been at Nordjyske for over 25 years.

Reviewer: ```suggestion
		\item A well-known journalist, who has been at Nordjyske for over 25 years.
```

	\end{itemize}
	\item System Administrator (9038 articles)
	\begin{itemize}
		\item We are not sure why this has been used. It could be a placeholder.
	\end{itemize}
\end{itemize}

> In \autoref{fig:auhtor_hist_after}, we see that the vast majority of authors have written under $2000$ articles within the three years. 

Reviewer 2: ```suggestion
In \autoref{fig:author_hist_before}, we see that the vast majority of authors have written under $2000$ articles within the three years. 
```
synes jeg viser det bedre

All authors and the number of articles they have written can be seen in \autoref{tab:author_table}.

> Unlike with Categories, the Author metadata field does not have a natural cut-off point, with a good amount of values in a specific lower range, followed by more evenly distributed numbers.

Reviewer: der er lidt mange taleord og er cut-off point ikke bare et threshold?

Author: jo

> Unlike with Categories, the Author metadata field does not have a natural cut-off point, with a good amount of values in a specific lower range, followed by more evenly distributed numbers.

Author: ```suggestion
Unlike with the category metadata, the author metadata does not have a natural cut-off point, with a good amount of values in a specific lower range, followed by more evenly distributed numbers.
```

Reviewer: ```suggestion
Unlike with the category metadata, the author metadata does not have a natural threshold, with authors that have values in a specific lower range, followed by more evenly distributed values.
```

> Unlike with Categories, the Author metadata field does not have a natural cut-off point, with a good amount of values in a specific lower range, followed by more evenly distributed numbers.

Reviewer 2: ```suggestion
Unlike with Categories, the Author metadata field does not have a natural cut-off point, with authors that have values in a specific lower range, followed by more evenly distributed numbers.
```

> Instead, the vast majority of the values fall in a lower range, meaning that setting too high a cut-off point, will result in removing a large portion of the data.

Reviewer: ```suggestion
Instead, the vast majority of the values fall in a lower range, meaning that setting the threshold too high will result in removing a large portion of the data.
```

> We instead choose a lower cut-off point, keeping most of the authors, except the ones that contained so few documents, that finding common topics would be inefficient.

Reviewer: ```suggestion
We instead choose a lower threshold, keeping most of the authors, except the ones that contained so few documents, that finding common topics would be inefficient.
```

> Authors, who have written less than $14$ documents ($0.01\%$ of the number of documents), are removed as part of preprocessing.

Author: ```suggestion
Authors, who have written less than $14$ documents (${\sim}0.01\%$ of the number of documents), are removed as part of preprocessing.
```

This removes $43$ out of $227$ authors, combining them into a single 'misc' author.
A total of $204$ documents are assigned to the 'misc' author.

\begin{figure*}[ht]
	\centering
	\begin{subfigure}{0.45\textwidth}
		\centering
		\includegraphics[width=\linewidth]{figures/author_hist2_before.png}
		\caption{Before preprocessing}
		\label{fig:author_hist_before}
	\end{subfigure}
	\begin{subfigure}{0.45\textwidth}
		\centering
		\includegraphics[width=\linewidth]{figures/author_hist2_14.png}
		\caption{After preprocessing}
		\label{fig:auhtor_hist_after}
	\end{subfigure}
	\caption{Histogram over amount authors who have written certain number of documents, before and after preprocessing.

> 		Authors on the x-axis are grouped into 50 buckets.}

Reviewer: ```suggestion
		Authors on the x-axis are grouped into 50 columns.}
```

	\label{fig:author_hist}
\end{figure*}

\begin{figure*}[ht]
	\centering
	\begin{subfigure}{0.45\textwidth}
		\centering
		\includegraphics[width=\linewidth]{figures/author_box_before.png}
		\caption{Before preprocessing}
		\label{fig:author_box_before}
	\end{subfigure}
	\begin{subfigure}{0.45\textwidth}
		\centering
		\includegraphics[width=\linewidth]{figures/author_box_14.png}
		\caption{After preprocessing}
		\label{fig:auhtor_box_after}
	\end{subfigure}
	\caption{Boxplot over the amount of documents written by authors.}
	\label{fig:author_box}
\end{figure*}

> Other than the 'System Administrator' author, there are a few authors that seem different since they do not have the name of a person.

Reviewer 2: ```suggestion
Other than the 'System Administrator' author, there are a few other authors that seem different since they do not have the name of a person.
```

> These are: 'SAXoTECH Systembruger' with 1 article, 'JSLbruger Nordjyske', 'testbruger', and 'Danske Fagmedier master' with 2 articles, and 'AAArtikler parkeret' with 51 articles.

Author: ```suggestion
These are: 'SAXoTECH Systembruger' with 1 article, 'JSLbruger Nordjyske', 'testbruger', and 'Danske Fagmedier master' with 2 articles each, and 'AAArtikler parkeret' with 51 articles.
```

After preprocessing, only 'AAArtikler parkeret' will not be put into the 'misc' author, since it has more than 13 articles.
From their names, they seem to be test users or authors that cover specific articles.
From looking at some articles these authors have written this seems to be the case since there are no obvious patterns in the articles written.
The name 'AAArtikler parkeret' also indicates that these articles, at least at some point, were unfinished and put under this author's name.

> These articles are kept in our dataset, even though we can not be certain of who wrote them.

Reviewer 2: ```suggestion
These articles are kept in our dataset, even though we can not be certain who wrote them.
```
